packages/perseus/src/widgets/radio/multiple-choice-component.new.tsx

```diff
@@ -35,65 +33,62 @@ export interface ChoiceType {
     showCorrectness: boolean;
     correct: boolean;
     isNoneOfTheAbove: boolean;
-    highlighted: boolean;
     previouslyAnswered: boolean;
     revealNoneOfTheAbove: boolean;
     disabled: boolean;
 }
 
-interface BaseRadioProps {
+/**
+ * Props for the MultipleChoiceComponent
+ */
+interface MultipleChoiceComponentProps {
```

Comment:
The `reviewMode` comment is super helpful! Would it be possible to add comments explaining what all the other props are for too?

Reply:
Hmm this might be worth a larger discussion about where we should be documenting our props, considering we're declaring them both here and in types. I added a comment for edit mode though since that was the other one that felt a little unclear for me.

Reply:
Oh! I tooootally missed that they're already in the types. You can ignore my comment then.